### PR TITLE
[infra] Add mio-circle05 support

### PR DESCRIPTION
**Pull request by jinevening**
This updates infra scripts to use mio-circle05.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/10629
Draft PR: https://github.com/Samsung/ONE/pull/10643

### CODE REVIEW

**Comment by jinevening** [comment]
I made a new preset following https://github.com/Samsung/ONE/issues/10553#issuecomment-1484483356.

File: infra/packaging/preset/20230413 [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  REQUIRED_UNITS+=("souschef")
+  REQUIRED_UNITS+=("safemain")
+  REQUIRED_UNITS+=("arser")
+  REQUIRED_UNITS+=("vconone")
+  # Hermes Logging Framework
+  REQUIRED_UNITS+=("hermes" "hermes-std")
+  # loco IR and related utilities
+  REQUIRED_UNITS+=("loco" "locop" "locomotiv" "logo-core" "logo")
+  # Flatbuffer I/O
+  REQUIRED_UNITS+=("mio-tflite280" "mio-circle04" "mio-circle05")
```
Comment (jinevening) [inline infra/packaging/preset/20230413:23]: This part is changed from the previous preset.

File: infra/packaging/preset/20230413_windows [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  REQUIRED_UNITS+=("souschef")
+  REQUIRED_UNITS+=("safemain")
+  REQUIRED_UNITS+=("arser")
+  REQUIRED_UNITS+=("vconone")
+  # Hermes Logging Framework
+  REQUIRED_UNITS+=("hermes" "hermes-std")
+  # loco IR and related utilities
+  REQUIRED_UNITS+=("loco" "locop" "locomotiv" "logo-core" "logo")
+  # Flatbuffer I/O
+  REQUIRED_UNITS+=("mio-tflite280" "mio-circle04" "mio-circle05")
```
Comment (jinevening) [inline infra/packaging/preset/20230413_windows:20]: ditto

**Comment by seanshpark** [comment]
there are multiple purpose of changes

1/ I hope you can change this to draft and split the changes to separate PR
and
2/ plz search inside `infra` folder with `mio-circle04` and add `mio-circle05` too
- `infra/scripts/compiler_modules.sh`
- `infra/scripts/docker_collect_nnpkg_resources.sh`

**Comment by jinevening** [comment]
> infra/scripts/compiler_modules.sh
infra/scripts/docker_collect_nnpkg_resources.sh

They are included in this PR.

**Comment by jinevening** [comment]
This draft is all merged.

**Pull request closed without merging**